### PR TITLE
Add recipe for hamronization

### PR DESCRIPTION
**Pull request by fmaguire**
This is a python module that implements tools to convert and summarize AMR
gene detection tool output using PHA4GE's hAMRonization output specification.

Project is hosted at github.com/pha4ge/hAMRonization and conda package and is built from PyPI

### CODE REVIEW

**Comment by fmaguire** [comment]
@BiocondaBot please add label

File: recipes/hamronization/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  number: 0
+  entry_points:
+    - hamronize = hAMRonization.hamronize:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pandas
+    - pip
+    - python
```
[the remark this answers is not in the document]
Comment (fmaguire) [reply recipes/hamronization/meta.yaml]: Ah good call.

File: recipes/hamronization/LICENSE.txt [hunk 2]
```diff
@@ -0,0 +1,166 @@
+                   GNU LESSER GENERAL PUBLIC LICENSE
```
[the remark this answers is not in the document]
Comment (fmaguire) [reply recipes/hamronization/LICENSE.txt]: I was using the conda skeleton from pypi and it added this field but I've removed it now.

**Review by thanhleviet — APPROVED** [review]
LGTM

**Comment by thanhleviet** [comment]
@BiocondaBot please merge

**Pull request merged**